jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/SubCaseServiceImplTest.java

```diff
@@ -513,7 +513,7 @@ public void testCaseWithSubCaseQuery() {
 
             caseService.triggerAdHocFragment(MID_CASE_ID, "Sub Case", null);
 
-            Collection<CaseInstance> caseInstances = caseRuntimeDataService.getCaseInstances(new QueryContext());
+            Collection<CaseInstance> caseInstances = caseRuntimeDataService.getCaseInstances(false,new QueryContext());
```

Comment:
not sure about this change? why not use `true` instead? or if that's not the aim of this test, why don't we keep it as it was originally?

Reply:
you are right, this is unneeded, so keeping the original